pkgs/development/python-modules/plugwise/default.nix

```diff
@@ -1,41 +1,42 @@
 { lib
-, buildPythonPackage
-, fetchFromGitHub
 , aiohttp
 , async-timeout
+, buildPythonPackage
 , crcmod
 , defusedxml
-, pyserial
-, pytz
-, python-dateutil
-, semver
+, fetchFromGitHub
 , jsonpickle
```

Comment:
Never referenced, but apparently needed, as imports are failing in the home-assistant test run.

Reply:
Okay, `jsonpickle` is in `requirements_test.txt`, I wish we could also have test dependencies autogenerated from the manifest, but there does not seem to be a field for that upstream.